src/components/Comments.jsx

```diff
@@ -37,21 +38,19 @@ const Comments = ({article_id}) => {
 
         <div>
 
-          
         {comments.map((comment) => {
 
             return <div className="comment-card" key={comment.comment_id}>
 
                 <h4>{comment.author}</h4>
                 <p>{comment.body}</p>
                 <p>votes: {comment.votes}</p>
-
+                
```

Comment:
I would prefer it if the comments were ordered by date DESC

Also i would put the posting inputs at the top of the comments list